src/mm-test-form-view/mm-test-form-view.html

```diff
@@ -0,0 +1,151 @@
+<!--
```

Comment:
@jcmoore @dlasky it's undocumented and not added to `strand.html`. I think it's essential to keep this for dogfooding/ fine-tuning, since the form and repeater will almost always be nested in a custom view.